src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        String colorName = Color.values()[index].name();
+        return colorName;
```

Comment:
There is no real value in creating variable to be used instantly and only once
```suggestion
        return Color.values()[index].name();
```